README.md

```diff
@@ -11,7 +11,8 @@
 
 # Nitric Java SDK
 The Java SDK supports the use of the cloud-portable [Nitric](https://nitric.io) framework with Java 11.
-> The SDK is in early stage development and APIs and interfaces are still subject to breaking changes
+
+> The SDK is available as early access and interfaces may still subject to breaking changes.
```

Comment:
"...interfaces  ["are" | "are still" | "may still be"] subject to breaking changes".

Reply:
Done